packages/ui/src/components/Visualization/ContextToolbar/ExportDocument/EntitiesMenu.scss

```diff
@@ -0,0 +1,7 @@
+.pf-v6-c-menu.pf-m-scrollable .pf-v6-c-menu__content {
+  overflow: auto;
+}
+
+.pf-v6-c-menu.pf-m-scrollable {
+  --pf-v6-c-menu--m-scrollable__content--MaxHeight: 36rem;
+}
```

Comment:
Are you facing an issue around the styling? Maybe we could find another way to avoid overriding patternfly classes

Reply:
For this specific one, we can remove it, not mandatory. I just wanted to make the dropdown taller to show more options at once. I always feel patternfly default is too short

The other one above, `overflow: auto`  is to put a scrollbar for the dropdown options. We can change it to be a custom class, but it needs to drill down some HTML tags in css as it's not the top one generated from patternfly component

Reply:
I guess it's "which is less ugly" thingy, both are susceptible for the change

Reply:
mmm, I see, I'll put a reminder so we can look at it next week, maybe there are other ways to influence the height.

Reply:
Nevermind, I tried again and found the spots custom css class works